src/components/icon-search-popover/index.js

```diff
@@ -243,13 +249,24 @@ const IconSearchPopover = props => {
 				</div>
 				<div className="ugb-icon-popover__iconlist">
 					{ isBusy && <Spinner /> }
-					{ ! isBusy && results.map( ( { prefix, iconName }, i ) => {
+					{ ! isBusy && applyFilters( 'stackable.global-settings.inspector.icon-library.icons', null, {
```

Comment:
This is a bit hard to read, I would like to suggest moving the `applyFilters` outside this area and change this to a component instead.

For example, somewhere near the top of the component:

```js
const IconLibraryIcons = applyFilters( 'stackable.global-settings.inspector.icon-library.icons', Fragment )
```

Then in this component:

```jsx
{ ! isBusy && (
	<IconLibraryIcons
		icons={ results.iconLibrary }
		onChange={ props.onChange }
		onClose={ props.onClose }
	/>
) }
```

you can add an `isPro` constant check in there as well to make sure it doesn't render if not in the premium version